tensorflow/python/keras/saving/save_test.py

```diff
@@ -83,6 +86,19 @@ def test_save_tf(self):
     save.save_model(self.subclassed_model, path, save_format='tf')
     self.assert_saved_model(path)
 
+  @test_util.run_v2_only
+  def test_save_load_tf_string(self):
+    path = os.path.join(self.get_temp_dir(), 'model')
+    save.save_model(self.model, path, save_format='tf')
+    save.load_model(path)
+
+  @test_util.run_v2_only
+  def test_save_load_tf_pathlib(self):
+    if sys.version >= "3.4":
+      path = pathlib.Path(self.get_temp_dir()) / 'model'
+      save.save_model(self.model, path, save_format='tf')
+      save.load_model(path)
```

Comment:
`ERROR: test_load_tf_pathlib (__main__.TestSaveModel)
test_load_tf_pathlib (__main__.TestSaveModel)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tensorflow/python/keras/save_test.runfiles/org_tensorflow/tensorflow/python/framework/test_util.py", line 1353, in decorated
    return f(self, *args, **kwargs)
  File "/tensorflow/python/keras/save_test.runfiles/org_tensorflow/tensorflow/python/keras/saving/save_test.py", line 100, in test_load_tf_pathlib
    save.load_model(pathlib.Path(path))
  File "tensorflow/python/keras/save_test.runfiles/org_tensorflow/tensorflow/python/keras/saving/save.py", line 155, in load_model
    loader_impl.parse_saved_model(filepath)
  File "tensorflow/python/keras/save_test.runfiles/org_tensorflow/tensorflow/python/saved_model/loader_impl.py", line 86, in parse_saved_model
    compat.as_bytes(export_dir),
  File "tensorflow/python/keras/save_test.runfiles/org_tensorflow/tensorflow/python/util/compat.py", line 87, in as_bytes
    (bytes_or_text,))
TypeError: Expected binary or unicode string, got PosixPath('/model')`

Reply:
can you please check above error ?

Reply:
That error is what I fixed in the last commit. Note the `line 100, in test_load_tf_pathlib` part; that test function was renamed to `test_save_load_tf_pathlib` in the last commit, so that traceback can't be from the most recent version.

The remaining CI failures don't appear to have anything to do with the changes in this pull request.

Reply:
sorry i did not check earlier updates , thank you